### PR TITLE
Add --testnet and --mainnet param option

### PR DESCRIPTION
**Pull request by Liaojinghui**
#413

### CODE REVIEW

File: neo-cli/Shell/MainService.cs [hunk 1]
```diff
@@ -1020,6 +1020,18 @@ protected internal override void OnStart(string[] args)
                     case "-r":
                         useRPC = true;
                         break;
+                    case "/testnet":
+                    case "--testnet":
+                    case "-t":
+                        File.Copy("config_testnet.json", "config.json", true);
+                        File.Copy("protocol_testnet.json", "protocol.json", true);
+                        break;
```
Comment (shargon) [inline neo-cli/Shell/MainService.cs]: Should be made without copy the files, using https://github.com/neo-project/neo/pull/959

**Comment by shargon** [comment]
@Liaojinghui please review this PR https://github.com/Liaojinghui/neo-cli/pull/1

**Comment by shargon** [comment]
This is for neo3, so there are none older

**Comment by Liaojinghui** [comment]
Cool

File: neo-cli/Shell/MainService.cs [hunk 2]
```diff
@@ -1020,6 +1021,18 @@ protected internal override void OnStart(string[] args)
                     case "-r":
                         useRPC = true;
                         break;
+                    case "/testnet":
+                    case "--testnet":
+                    case "-t":
+                        Settings.Initialize(new ConfigurationBuilder().AddJsonFile("config_testnet.json").Build());
```
Comment (erikzhang) [inline neo-cli/Shell/MainService.cs]: Not "config.testnet.json"?

**Comment by vncoelho** [comment]
@Liaojinghui, can you merge with the latest master?

**Comment by Liaojinghui** [comment]
> @Liaojinghui, can you merge with the latest master?

I think it is already based on the latest master or could be merged without any conflict.

**Comment by vncoelho** [comment]
Can you update to the latest daily package? https://www.myget.org/feed/neo/package/nuget/Neo
The latest is 132. Then, it will probably compile here.

**Comment by Liaojinghui** [comment]
> Can you update to the latest daily package? https://www.myget.org/feed/neo/package/nuget/Neo
> The latest is 132. Then, it will probably compile here.

Well, feels like neo-cli has a lot of work to be done to match the 123

**Comment by vncoelho** [comment]
I mean, we need to update the nuget package, I will PR your branch.

**Comment by vncoelho** [comment]
I think it will compile now, check it out @Liaojinghui.

**Comment by Liaojinghui** [comment]
> I mean, we need to update the nuget package, I will PR your branch.

May I ask you how did you make this neo-cli work on the latest NEO3.0 core?

**Comment by vncoelho** [comment]
It is working now.

**Comment by Liaojinghui** [comment]
> It is working now with PR #414.
> In fact, this should be done in the top of it.

No clue how did you manage to get rid of the errors like `Can't find the definition of **FillTransaction** in Wallet`

**Comment by Liaojinghui** [comment]
> It is working now with PR #414.
> In fact, this should be done in the top of it.

Always get the same error you met before https://github.com/neo-project/neo/pull/959#issuecomment-515237295

**Comment by vncoelho** [comment]
@Liaojinghui, no worries, it is fixed on that Branch. Let's just wait.

**Comment by shargon** [comment]
I will test it, and then i will approve it

**Pull request merged**